app/services/reports/mis/application_detail_csv_line.rb

```diff
@@ -442,6 +444,10 @@ def linked_applications
         end
       end
 
+      def home_address
+        @line << yesno(laa.applicant.no_fixed_residence?)
```

Comment:
This looks like it will be "Yes" for " Home address" column when `no_fixed_residence?` is true. Is that what is needed?

Reply:
I think it's right, let's see what Mike thinks when testing! 😄